File: typescript-examples/starter-auth/api/sample.ts
import { BrowserContext, Page } from "playwright";
import { goToUrl } from "@intuned/browser";


interface Params {
}



export default async function handler(
  params: Params,
  page: Page,
  context: BrowserContext
) {
  const sandboxedUrl = "https://sandbox.intuned.dev/consultations-auth/book";

  await goToUrl({
    page: page,
    url: sandboxedUrl,
  });
  

  return {
  };
}

File: python-examples/starter-auth/api/sample.py
from typing import TypedDict

from intuned_browser import go_to_url
from playwright.async_api import Page


class Params(TypedDict):
    pass


async def automation(page: Page, params: Params | None = None, **_kwargs):
    sandboxed_url = "https://sandbox.intuned.dev/consultations-auth/book"

    await go_to_url(page=page, url=sandboxed_url)

    return {}
